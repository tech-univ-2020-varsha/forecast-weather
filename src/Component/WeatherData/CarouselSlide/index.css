.carousel{
    display: flex;
    flex-direction: column;
    padding: 1em;
    border: solid white;
    margin: 1em;
    justify-content: center;
}

.carouselData{
    align-items: center;
    color: rgba(27, 14, 14, 0.849);
    padding: 0.2em;
}