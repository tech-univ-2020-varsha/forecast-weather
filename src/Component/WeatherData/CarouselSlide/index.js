import React from 'react';
import './index.css';

const CarouselSlide = ({ temp, pressure, humidity }) => (
  <div className="carousel">
    <div className="carouselData">
      T:
      {' '}
      {temp}
    </div>
    <div className="carouselData">
      P:
      {' '}
      {pressure}
    </div>
    <div className="carouselData">
      H:
      {' '}
      {humidity}
    </div>

  </div>
);


export default CarouselSlide;
