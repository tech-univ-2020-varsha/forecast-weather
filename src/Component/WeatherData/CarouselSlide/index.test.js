import { render } from '@testing-library/react';
import React from 'react';
import CarouselSlide from '.';

describe('the carousel slide function', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<CarouselSlide temp={898} pressure={9898} humidity={9998} />);
    expect(asFragment).toMatchSnapshot();
  });
});
