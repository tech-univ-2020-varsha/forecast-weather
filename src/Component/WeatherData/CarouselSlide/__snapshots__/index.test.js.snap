// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the carousel slide function should render correctly 1`] = `[Function]`;
